scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -146,6 +146,9 @@ def __init__(self, **kwargs):
         self.notification_recipient_email = (
             kwargs.get('notification_recipient_email'))
         self.gsuite_superadmin_email = kwargs.get('gsuite_superadmin_email')
+        self.host_project_id = kwargs.get('host_project', self.project_id)
+        self.vpc_name = kwargs.get('vpc') or 'default'
```

Comment:
The get() would support a default option:
```
self.vpc_name = kwargs.get('vpc', 'default')
```

Reply:
I completely agree but this follows convention with the rest of this file (see line 135 for example). Happy to change it still.